hack/benchmark/e2e/agent/ngt/helper_test.go

```diff
@@ -0,0 +1,60 @@
+package ngt
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
Could you run `make license` to add license header?